packages/models-library/tests/data/metadata-ascent-1.3.2.json

```diff
@@ -0,0 +1,139 @@
+{
+  "authors": [
+    {
+      "name": "Katie Zhuang",
+      "email": "zhuang@test.it",
+      "affiliation": "IT'IS Foundation"
+    }
+  ],
+  "contact": "zhuang@test.it",
+  "description": "ASCENT is an open source platform for simulating peripheral nerve stimulation. To download the software, visit the [ASCENT GitHub repository](https://github.com/wmglab-duke/ascent). This implementation uses a limited set of user-defined parameters to run the pipeline and will output the results of the NEURON simulations * Musselman ED, Cariello JE, Grill WM, Pelot NA. ASCENT (Automated Simulations to Characterize Electrical Nerve Thresholds): A Pipeline for Sample-Specific Computational Modeling of Electrical Stimulation of Peripheral Nerves. PLoS Comput Biol [Internet]. 2021; Available from: https://doi.org/10.1371/journal.pcbi.1009285 * Musselman ED, Cariello JE, Grill WM, Pelot NA. ASCENT (Automated Simulations to Characterize Electrical Nerve Thresholds): A Pipeline for Sample-Specific Computational Modeling of Electrical Stimulation of Peripheral Nerves. PLoS Comput Biol [Internet]. 2021, DOI: 10.5281/zenodo.5500260\n",
+  "inputs": {
+    "input_1": {
+      "displayOrder": 1,
+      "label": "Nerve Morphology",
+      "description": "Choice of Vagus nerve morphology - either Rat VN or Human VN",
+      "type": "ref_contentSchema",
+      "contentSchema": {
+        "title": "Nerve Morphology",
+        "default": "Rat",
+        "enum": [
+          "Rat",
+          "Human"
+        ]
+      }
+    },
+    "input_2": {
+      "displayOrder": 2,
+      "label": "Cuff geometry",
+      "description": "45\u00b0 contact wrap monopolar cuff or 360\u00b0 contact wrap monopolar cuff or 45\u00b0 contact wrap, bipolar cuff or 360\u00b0 contact wrap, bipolar cuff \n",
+      "type": "ref_contentSchema",
+      "contentSchema": {
```

Comment:
So I have a question here, why are all the inputs contentSchema and not the outputs? or else why are there no "legacy" types in the inputs and only "legacy" types in the outputs?

Reply:
the outputs in this example are all files. For the moment, the file still uses the "legacy" type.